.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,7 +14,7 @@ Close #
 如果路由包含在文档中列出可以完全穷举的参数（例如分类），请依次全部列出。
 
 Please include route starts with /, with all required and optional parameters. Fail to comply will result in your pull request being closed automatically.
-```route
+```routes
```

Comment:
If you change it to `routes`, the example routes below will be picked up by the parser.

https://github.com/DIYgod/RSSHub/blob/5c0b7f24cc1ac381c54ee70986c9140ec022d23e/scripts/workflow/test-route/identify.js#L7

Reply:
Okay, but this may be misleading to the contributors. Maybe we can remove the example `route` part
````
```route
/some/route
/some/other/route
```
````
just in the template? Because it has been illustrated in https://github.com/DIYgod/RSSHub/discussions/8002, and the following
````
```routes
```
````
works well in the template.

Reply:
The instruction was removed in 9bc99cefb4d9491ed5268d57ab23207ff7876cb1 and added back in #8379.

The reason was
> Not every contributor needs to read through https://github.com/DIYgod/RSSHub/discussions/8002.

Reply:
It's ok. Thanks for your reply!